### PR TITLE
Article on WordPress on DreamCompute

### PR DESCRIPTION
**Pull request by Ipstenu**
Proposed KB article for using WordPress on DreamCompute.

I cede all grumpy ownership rights of this document. Edit and fix and play and tweak as needed to meet your formatical and contentical needs.

### CODE REVIEW

File: source/dreamcompute/tutorials/deploy-wordpress-on-dreamcompute.rst [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+    sudo chown -R wp_example:www-data /var/www/example.com/
+
+The reason we add the users to the www-data group is to allow Ubuntu to properly manage WordPress updates and images.
+
+Add SSH Access
+~~~~~~~~~~~~~~
+
+WordPress users often need SSH access in order to do extra configuration with WordPress. By default, this is disabled, so you will need to edit your config
+
+If you need SSH access
```
Comment (justinknierim) [inline source/dreamcompute/tutorials/deploy-wordpress-on-dreamcompute.rst]: is there some new line weirdness going on here perhaps?  Seems like "If you need SSH access" and the next line having "apparently not" are odd.

**Comment by justinknierim** [comment]
I think there are some lines that are longer than 80 chars, which "tox" might complain about too.  Can be fixed up pretty easy though.

**Comment by Ipstenu** [comment]
I fixed the lines and the weird SSH thing which was caused by brain derp :)

**Comment by squidboylan** [comment]
I'll fix the 80 char line length, other than that it looks good, thanks

**Pull request merged**